0010-zkapp-cli-lightnet-management.md

```diff
@@ -0,0 +1,100 @@
+# RFC for zkapp-cli enhancements to provide the local lightnet networks management
+
+Introduce new sub-commands to the existing zkapp CLI application to streamline the local lightnet networks management for development and testing purposes.
+
+## Summary
+
+We aim to extend the current zkapp CLI application by introducing the `lightnet` sub-command suite. This suite will centralize and simplify Docker containers and provided tools management tasks for local zkApps development and testing, enhancing user and developer experience.
+
+## Motivation
+
+The primary motivation is to simplify the local zkApps development and testing process by offering intuitive CLI commands.
+
+Expected Outcomes:
+
+- A streamlined workflow for managing local lightnet network [Docker containers](https://hub.docker.com/r/o1labs/mina-local-network) and tools provided by such containers.
+- Reduced complexity and better DX, eliminating manual 3rd party tools management.
+
+## Detailed design
+
+### zkapp CLI Interface Enhancements
+
+Under the primary command `zk`, the new `lightnet` sub-commands will be introduced:
+
+#### Help
+
+- **Command**: `zk lightnet`
+- **Output**: Help message with description of available sub-commands and their options.
+
+#### Start lightnet network
+
+- **Command**: `zk lightnet start <options>`
+- **Function**: Checks the env., starts the local lightnet Docker container with reasonable default configuration options and waits until the network sync. Repeating the command while network is up and running should not start a new container, but rather check the status of the existing one.
```

Comment:
What will be checked in the env. ? Is the env. configured by the options entered in `zk lightnet start <options>`  or from somewhere else?

Reply:
Here by "checks the env." I was only thinking about:
- check that docker engine installed and accessible by user
- check internet connectivity to make it possible download the image
- check that system has enough space to unpack the image and start the container